docs/eep.schema

```diff
@@ -94,8 +94,8 @@
                                 "gpio": {
                                     "description": "GPIO or pin number",
                                     "type": "integer",
-                                    "minimum": 2,
```

Comment:
The GPIOs 0 and 1 are excluded as they are configured by the dt-blob.bin. Otherwise the bootloader wouldn't be able to read the hat eeprom.
https://github.com/RevolutionPi/linux/blob/1bbba2d8bff3358a9ab6cb9adc0789ac27408a94/arch/arm/boot/dts/overlays/revpi-dt-blob-overlay.dts#L39
https://github.com/RevolutionPi/linux/blob/1bbba2d8bff3358a9ab6cb9adc0789ac27408a94/arch/arm/boot/dts/overlays/revpi-dt-blob-overlay.dts#L19